frontend/src/app/features/bim/ifc_models/bcf/list-container/bcf-list-container.component.html

```diff
@@ -1,37 +1,39 @@
-<div class="result-overlay"
-     *ngIf="(showResultOverlay$ | async) && showTableView"></div>
+<div class="op-bcf-list-container">
+     <div class="op-bcf-list-container--result-overlay"
+          *ngIf="(showResultOverlay$ | async) && showTableView"></div>
 
-<!-- TABLE + TIMELINE horizontal split -->
-<div class="work-packages-split-view--tabletimeline-container"
-     [ngClass]="{ '-with-resizer': showResizerInCardView() }"
-     *ngIf="tableInformationLoaded && showTableView">
-  <wp-resizer elementClass="work-packages-partitioned-page--content-right"
-              localStorageKey="openProject-splitViewFlexBasis">
-  </wp-resizer>
+     <!-- TABLE + TIMELINE horizontal split -->
+     <div class="op-bcf-list-container--work-packages-split-view--tabletimeline-container"
+          [ngClass]="{ '-with-resizer': showResizerInCardView() }"
```

Comment:
Please use the underscore notation instead of a separated modifier class (so something like `op-bcf-list{..}.tabletimeline-container_with-resizer`)